### ISSUE TITLE
adddata to map using ArrowLoader failed  to  bind  on  map

### ISSUE THREAD

**Issue by vikashsingh009** [visgl/loaders.gl#1032]
install arrow loader using npm
import Arrow Loader
here is the code
```
async _loadArrowData() {
     var url= 'https://gist.githubusercontent.com/TheNeuralBit/64d8cc13050c9b5743281dcf66059de5/raw/c146baf28a8e78cfe982c6ab5015207c4cbd84e3/scrabble.arrow'

    const data =  await parse(fetchFile(url), ArrowLoader, {worker: false});
    console.log("data",data)
    console.log("dataType",data.type)
    // load arrow
    this.props.dispatch(
      addDataToMap({
        datasets: [
          {
            info: {label: 'Pipeline Arrow', id: 'pipe2071'},
            data: processRowObject(data)
          },
          //  {
          //    info: {label: 'SF Zip Geo', id: 'sf-zip-geo'},
          //    data: processGeojson(sampleGeojson)
          //  }
        ],
        options: {
          keepExistingConfig: true
        },
        //config: sampleGeojsonConfig
      })
    );
  }
```

here  is  the  issues  link
https://github.com/keplergl/kepler.gl/issues/1332

**Comment by ibgreen** [comment]
Solution: add a loader option so that `parse(ArrowLoader)` can return a standard JS table (convert columns to rows before returning).

We have some of this code for the `parseInBatches` but it should work for all cases.

**Comment by vikashsingh009** [comment]
Thanks @ibgreen  for  you  solution .it works  for me 
below  code  changes 

```
const values = [];
   
    for await (const batch of await loadInBatches(url, ArrowLoader)) {
      for (const row of batch) {
        values.push(batch);
      }
    }
```

**Comment by vdunaev** [comment]
Hi everyone! I've published 2.3.7 patch with this fix
